lib/ext/heroku/command/addons.rb

```diff
@@ -225,6 +225,14 @@ def add
       end
     end
 
+    # addons:add ADDON
+    #
+    # WARNING: `heroku addons:add has been deprecated. Please use `heroku addons:create` instead.
```

Comment:
I think comment == documentation appears if comments exist. So we may want to just nuke the comment altogether.